dvc/remote/azure.py

```diff
@@ -81,6 +81,15 @@ def blob_service(self):
             blob_service.create_container(self.path_info.bucket)
         return blob_service
 
+    def get_content_md5(self, path_info):
+        content_md5 = self.blob_service.get_blob_properties(
+            path_info.bucket, path_info.path
+        ).properties.content_settings.content_md5
```

Comment:
Does it stay true for multipart objects?

Reply:
Also, does the user set it or is it automatically set by azure?

Reply:
We need very good tests for this to get it accepted, as otherwise it might be too dangerous to use.

Reply:
For potential future reference: `Content-MD5` appears to not behave as specified in the [documentation of the REST API](https://docs.microsoft.com/en-us/rest/api/storageservices/put-blob). Namely, large files (I tested with a 4GB file) do not get assigned a `Content-MD5` property automatically. Hence, I've changed it back to use the etag instead

(see also: https://discordapp.com/channels/485586884165107732/565699007037571084/713462619469643796)